Source/DafnyCore/Rewriters/OpaqueMemberRewriter.cs

```diff
@@ -80,7 +80,8 @@ public OpaqueMemberRewriter(ErrorReporter reporter)
     Contract.Requires(c != null);
     var newDecls = new List<MemberDecl>();
     foreach (var member in c.Members.Where(member => member is Function or ConstantField)) {
-      if (!Attributes.Contains(member.Attributes, "opaque") && !member.IsOpaque) {
+      if (!Attributes.Contains(member.Attributes, "opaque") && !member.IsOpaque &&
```

Comment:
Can you please put this new conjunct in a separate function since it starts to be more complicated?